tools/send-to-caas/send-utils.js

```diff
@@ -253,7 +253,14 @@ const getDateProp = (dateStr, errorMsg) => {
 
 const getOrigin = () => {
   const origin = getConfig().project || getConfig().repo;
-  if (origin) return origin;
+  const mappings = {
+    cc: 'hawks',
+    dc: 'doccloud',
+  };
+
+  if (mappings[origin]) {
+    return mappings[origin] || origin;
```

Comment:
Since you're checking that `mappings[origin]` exist in the `if` statement, this will always return `mappings[origin]` and never `origin`. I think the `if` statement can be removed entirely.